[ADD] sales: add Pricer discounts configuration (#11527)
odoo/documentation: content/applications/sales/point_of_sale/pricing/electronic_labels.rst

.. warning::
   If a request sent to Pricer fails, Odoo still considers that the product has been updated. In
   that case, we recommend forcing the update of all tags.

> 

Reviewer: Remove this space

Discounts
---------

> To display a discount on a Pricer Tag, you have to link a pricelist to the :guilabel:`product variant`
> associated with the tag.

Reviewer: - I've added a link to the pricelist doc
- I removed the guilabel since it's the concept and not something displayed on screen at that point
- + 100th character
```suggestion
To display a discount on a Pricer Tag, you need to link a :doc:`pricelist <pricelists>` to the
product variant associated with the tag.
```

> 

Reviewer: I suggest "explaining" the steps before #. 
```suggestion

Then, open the product variant form:

```

> #. Go to :guilabel:`Product Variants` tab,

Reviewer: It's better to use a menuselection when there's multiple clicks to getting where you need to
Also, to stay consistent with the punctuation above, I've changed the "," to "."
```suggestion
#. Go to :menuselection:`Point of Sale --> Products --> Product Variants`.
```

> #. Select the product you want to apply a discount to,

Reviewer: ```suggestion
#. Select the product you want to apply a discount to.
```

> #. In the product form, look for :guilabel:`Pricer Sales Pricelist` field,
> #. Select the pricelist you want from the list,

Reviewer: ```suggestion

And set the desired pricelist:

#. Go to the :guilabel:`General Information` tab.
#. Select a pricelist in the :guilabel:`Pricer Sales Pricelist` field.
```

> #. You can now see a new field :guilabel:`On Sale Price` that corresponds to the Sales Price with the discount

Reviewer: ```suggestion
Once a pricelist is set, the :guilabel:`On Sale Price` field appears, showing the :guilabel:`Sales
Price` with the discount applied.
```

   applied.

.. image:: electronic_labels/pricer-sales-pricelist.png
   :alt: Linking a pricelist to a product variant

> If you update your electronic labels, you should now see a *PROMO* tag appear, along with the old crossed-out
> price and the discounted price.

Reviewer: ```suggestion
After updating your electronic labels, a `PROMO` tag should appear on the label, displaying both the
old, crossed-out price and the discounted price.
```

> .. note::
>     For now, we don't handle pricelists that apply on quantities above one (i.e. discounts if one buys more
>     than one product), or pricelists based on formula *based on other pricelists*

Reviewer: ```suggestion
.. note::
   Currently, pricelists that offer discounts for purchasing multiple units or derive their prices
   from other pricelists are not supported.
```
